charts/shoot-core/charts/csi-packet/templates/setup.yaml

```diff
@@ -0,0 +1,41 @@
+kind: ClusterRole
```

Comment:
Not exactly sure, but probably you might also need to have some PSP (see https://github.com/gardener/gardener/tree/master/charts/shoot-core/charts/csi-alicloud/templates/psp).

Reply:
Yeah that probably is necessary to give it the right permissions. Adding.

Reply:
Done